splunk/dasg_metrics_dashboard.xml

```diff
@@ -184,11 +182,60 @@
       </chart>
     </panel>
   </row>
+  
   <row>
     <panel>
       <html>
        <u1> <h2>
-            <b>Metrics Gathered from FHIR Resource Logging Events</b>
+            <b>PROD Third party apps beneficiary counts</b>
+          </h2>
+        </u1>
+     </html>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <title>1. Apps served with MORE than 25 beneficiaries</title>
+      <single>
+        <search>
+          <done>
+            <set token="tokFhirV1PatientOKCount">$result.count$</set>
+          </done>
+           <!-- message.group_timestamp="2021-07-14T07:00:54+00:00" -->
+          <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=* env=prod | search "message.type"="global_state_metrics_per_app" "message.group_timestamp"=$tokGroupTimestamp|s$ message.active=True message.real_bene_cnt>25 | stats count</query>
+          <earliest>$t_local.earliest$</earliest>
+          <latest>$t_local.latest$</latest>
+        </search>
+        <option name="drilldown">all</option>
+        <option name="height">226</option>
+        <option name="refresh.display">progressbar</option>
+      </single>
+    </panel>
+    <panel>
+      <title>2. Apps served with LESS than 25 beneficiaries</title>
+      <single>
+        <search>
+          <done>
+            <set token="tokFhirV1PatientOKCount">$result.count$</set>
+          </done>
+           <!-- message.group_timestamp="2021-07-14T07:00:54+00:00" -->
+          <query>index=bluebutton source="/var/log/pyapps/perf_mon.log*" host=* env=prod | search "message.type"="global_state_metrics_per_app" "message.group_timestamp"=$tokGroupTimestamp|s$ message.active=True message.real_bene_cnt&lt;25 | stats count</query>
```

Comment:
@oragame This should be edited for "Apps served with 25 or fewer/less beneficiaries".   And the search comparison would be "<=".

Might be good to add another panel with the grand total too?

Reply:
Good idea, I have added the total count and updated the operator to "<=" for that panel.